contracts/upgradeable_contracts/arbitrary_message/BasicAMB.sol

```diff
@@ -47,27 +52,27 @@ contract BasicAMB is BasicBridge {
     }
 
     function setSubsidizedModeForHomeToForeign() external onlyOwner {
-        bytesStorage[HOME_TO_FOREIGN_MODE] = SUBSIDIZED_MODE;
+        uintStorage[HOME_TO_FOREIGN_MODE] = SUBSIDIZED_MODE;
```

Comment:
does it make sense to raise the event `HomeToForeignModeChanged` here?

Reply:
Yes, updated the setter for this mode to emit the event https://github.com/poanetwork/poa-bridge-contracts/pull/77/commits/f9efd5239261b48d4842d1b81e96fbfcd028b98a